docs/readme_template.md

```diff
@@ -0,0 +1,24 @@
+# MOD2 // PROJECT NAME
```

Comment:
I like this layout for the README template so we aren't cramming to come out with an overview at the very end of the project week